src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
+    # We are going to prepend this to compilation tool invocations, so we need to take
+    # the specified host triple and at a '-'
+    HOST=`expr "X$1" : '[^=]*=\(.*\)'`-;
+    shift ;;
+  --fst-version=*)
+    OPENFST_VER=`expr "X$1" : '[^=]*=\(.*\)'`;
+    shift;;
+  --target-arch=*)
+    TARGET_ARCH=`expr "X$1" : '[^=]*=\(.*\)'`;
+    shift ;;
+  --android)
+    threaded_math=false;
+    static_math=true;
+    static_fst=true;
+    dynamic_kaldi=false;
+    MATHLIB='OPENBLAS';
+    TARGET_ARCH='arm';
+    android=true;
+    shift;;
+  --toolchain-includes=*)
```

Comment:
As it is this option name does not signal it has to do with android. Maybe we can call this "--android-incdir" so it is more consistent with the other option names.